From 83d3cdadb3b6e92289315f72d1a18613deb4e9f5 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Mon, 26 Sep 2022 19:32:43 +0300
Subject: [PATCH] [SW-105911] fuse: remove initialization of spi die offset

Since spi offsets will be part of persistent dccm which will be
retained across non-cold reset, this patch removes initialization of
spi offset.

Change-Id: I464a809a44060a6e4b16236fa7c8c5a7905a876f
---

diff --git a/app/arcboot_rom.c b/app/arcboot_rom.c
index 55dc085..6f119ee 100644
--- a/app/arcboot_rom.c
+++ b/app/arcboot_rom.c
@@ -27,7 +27,6 @@
 	.curr_img = 0,
 	.die_id = 0,
 	.sku = 0,
-	.d0_fuse_spi_offset = 0x0,
 	.fuse_spi = false,
 };
 
